### DISCUSSION TITLE
Find all references

### DISCUSSION THREAD

**Discussion by fr43nk** [clangd/clangd#2114] [General]
Hi all,

using this extension in vscode is very helpful and fast. Only one thing does not work: "Find all References" this does only list items in the current open editors, not the whole workspace.
I think one problem might be, that I do not use CMake as my tool chain. In the configuration I use the parameter fallbackFlags to provide the include path and define information for the language server.

Can anyone help?

Best regards,
Frank

**Comment by HighCommander4** [comment]
Please see https://clangd.llvm.org/faq#why-does-clangd-not-return-all-references-for-a-symbol and https://clangd.llvm.org/installation#compile_commandsjson. CMake is not required but generating a `compile_commands.json` file by some means, is.
